### PR TITLE
Coverity scan fixes

### PR DESCRIPTION
**Pull request by lpechacek**

### CODE REVIEW

File: libqf/libqfcore/src/sql/connection.cpp [hunk 1]
```diff
@@ -174,7 +174,7 @@ static QVariant sqlite_set_pragma(QSqlQuery &q, const QString &pragma_key, const
 		qfError() << QString("SQL Error\nquery: %1;").arg(qs);
 		return QVariant();
 	}
-	q.next();
+	QF_CHECK(q.next(), "SQL PRAGMA query is supposed to return data but it hasn't");
```
Comment (fvacek) [inline libqf/libqfcore/src/sql/connection.cpp]: please use rather
```c++
if(q.next()) {
...
}
else {
  qfError() << "SQL PRAGMA query is supposed to return data but it hasn't";
}
```

File: quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp [hunk 2]
```diff
@@ -106,13 +106,13 @@ void EventConfig::save(const QString &path_to_save)
 			val_str = val.toString();
 		q_up.bindValue(":key", key);
 		q_up.bindValue(":val", val);
-		q_up.exec();
+		q_up.exec(qf::core::Exception::Throw);
```
Comment (fvacek) [inline quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp]: `q_up.exec(!qf::core::Exception::Throw);` 

is intention here, there is no try-catch block in function call stack, so I prefer not crashing app with error in log in this case

If we want to enable throw exception here, we should change more code in other places

File: libqf/libqfcore/src/sql/connection.cpp (hunk 1 above)
Comment (lpechacek) [inline libqf/libqfcore/src/sql/connection.cpp]: OK, done.

File: quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp (hunk 2 above)
Comment (lpechacek) [inline quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp]: Understood.  A few lines above there is
```
q_up.prepare("UPDATE config SET cvalue=:val WHERE ckey=:key", qf::core::Exception::Throw);
...
q_ins.prepare("INSERT INTO config (ckey, cvalue, ctype) VALUES (:key, :val, :type)", qf::core::Exception::Throw);
```
Should these occurrences of exception throwing be replaced with return value checking too?

Comment (fvacek) [inline quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp]: Maybe try-catch around all the sql calls will be better. I agree that sql calls should be handled consistently

**Comment by lpechacek** [comment]
take 2 - Patches updated to include feedback.

**Pull request merged**